apps/fluent-tester/src/FluentTester/theme/ThemePickers.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-import { View, Picker, StyleSheet } from 'react-native';
```

Comment:
I noticed in ThemePicker.android.tsx we refer to `PickerPropsAndroid` from 'react-native`. We probably want to remove that reference and replace it with something else.

Reply:
I didn't notice that, thanks! I've replaced it with the picker props.